.appveyor.yml

```diff
@@ -6,7 +6,13 @@ init:
   - ps: $PSVersionTable
 
 build_script:
-  - ps: ./bootstrap.ps1
-
-test_script:
-  - ps: chef exec inspec exec test/integration/**/*.rb --format documentation
+  - ps: . { (New-Object System.Net.WebClient).DownloadString('https://omnitruck.chef.io/install.ps1') } | iex; install -project 'chefdk' -version '0.17.17'
+  - ps: $env:path += ';C:\opscode\chefdk\bin;'
+  - ps: chef exec bundle install
+  - ps: chef exec bundle exec rake
+  - ps: chef gem build ./workup.gemspec
+  - ps: chef gem install ./workup-0.1.0.gem --bindir 'C:\opscode\chefdk\bin'
+  - ps: mkdir ~/.workup
+  - ps: cp ./files/Policyfile.rb ~/.workup/Policyfile.rb
```

Comment:
I would just make a folder in the test folder to mock this stuff and move the entire folder to ~/.workup